genesyscloud/routing_queue/data_source_genesyscloud_routing_queue.go

```diff
@@ -42,20 +43,34 @@ func dataSourceRoutingQueueRead(ctx context.Context, d *schema.ResourceData, m i
 // hydrateRoutingQueueCacheFn for hydrating the cache with Genesys Cloud routing queues using the SDK
 func hydrateRoutingQueueCacheFn(c *rc.DataSourceCache, ctx context.Context) error {
 	proxy := GetRoutingQueueProxy(c.ClientConfig)
+	var allQueues []platformclientv2.Queue
 
 	log.Printf("Hydrating cache for data source %s", ResourceType)
 
-	allQueues, resp, err := proxy.GetAllRoutingQueues(ctx, "")
+	queues, resp, err := proxy.GetAllRoutingQueues(ctx, "", false)
 	if err != nil {
 		return fmt.Errorf("failed to get routing queues. Error: %s | API Response: %s", err.Error(), resp.String())
 	}
 
-	if allQueues == nil || len(*allQueues) == 0 {
+	if queues != nil || len(*queues) != 0 {
+		allQueues = append(allQueues, *queues...)
+	}
+
+	trueQueues, resp, err := proxy.GetAllRoutingQueues(ctx, "", true)
```

Comment:
Shouldnt we be logging the response in our standard logging format so it is in JSON